packages/headless/src/controllers/insight/category-facet/headless-insight-category-facet.ts

```diff
@@ -0,0 +1,233 @@
+import {
+  buildCoreCategoryFacet,
+  CategoryFacetSearch,
+  CategoryFacetValue,
+  CoreCategoryFacet,
+  CoreCategoryFacetState,
+} from '../../core/facets/category-facet/headless-core-category-facet';
+import {CategoryFacetOptions} from '../../core/facets/category-facet/headless-core-category-facet-options';
+import {CategoryFacetSortCriterion} from '../../../features/facets/category-facet-set/interfaces/request';
+import {InsightEngine} from '../../../insight.index';
```

Comment:
You should not be referencing the `insight.index` file. You should point to the `insight-engine.ts` file instead.